src/HtmlProcessor/AbstractHtmlProcessor.php

```diff
@@ -103,9 +103,19 @@ private function setHtml(string $html)
      * Provides access to the internal DOMDocument representation of the HTML in its current state.
      *
      * @return \DOMDocument
+     *
+     * @throws \UnexpectedValueException
      */
     public function getDomDocument(): \DOMDocument
     {
+        if (null === $this->domDocument) {
```

Comment:
Yoda not like this :)

Reply:
looks like this can't be cleanly cherry-picked, so it'll be addressed directly.

Reply:
resolved in 451e379